absearch/tests/test_server.py

```diff
@@ -59,35 +59,6 @@ def test_version():
     version = app.get('/__version__').json
     assert version['project'] == 'absearch'
 
-
-def test_swagger():
-    app = get_app()
-    # test the APIs
-    res = app.get('/__api__')
-
-    # make sure it's compliant
-    swagger_dict = yaml.load(res.body, Loader=yaml.FullLoader)
-    parser = SwaggerParser(swagger_dict=swagger_dict)
```

Comment:
This swagger parser dependency should not be useful anymore

Reply:
will do